envoy/network/dns.h

```diff
@@ -57,6 +57,10 @@ class ActiveDnsQuery {
 
   /** Return the DNS query traces. */
   virtual OptRef<const std::vector<Trace>> getTraces() PURE;
```

Comment:
Do we need this function still?

Reply:
It's used for testing. It's not that straightforward to assert the traces timestamps.

Reply:
do we need to assert on the timestamps?  Can't we just check the fields and leave the rest?

Reply:
Updated the code.